src/boxConfiguration/go.mod

```diff
@@ -0,0 +1,3 @@
+module github.com/th2-net/th2-box-template-go/src/boxConfiguration
```

Comment:
it is redundant module. The project should have only two modules:
th2-grpc and own in from root folder